parsl/app/futures.py

```diff
@@ -210,7 +210,7 @@ def testing_nonfuture():
         for item in nums:
             testfile.write("{0}\n".format(item))
 
-    foo = Future()
+    foo = Future()  # type: Future[str]
```

Comment:
Does `Future[str]` mean a future that'll resolve to a string ?

Reply:
yes